RealEstate/apps/core/templates/core/houseEval.html

```diff
@@ -0,0 +1,105 @@
+{% extends "core/base.html" %}
+{% load bootstrap3 %}
+{% load staticfiles %}
+
+{% block style %}
+<link rel="stylesheet" type="text/css" href="{% static "slide.css" %}">
+<style>
+.slider-selection {
+	background: #BABABA;
+}
+.vcenter {
+	display: inline-block;
+	vertical-align: middle;
+	float: none;
+	margin-left:-3px;
+}
+.alert-success {
+	margin-top: 30px;
+}
+ul {
+	margin-top:50px;
+}
+.inner-rating {
+	padding:20px 0;
+}
+</style>
+{% endblock %}
+
+{% block body %}
+	<h2><strong> Evaluation for {{house}}</strong> </h2>
+	<h4><em>{{house.address}}</em></h4>
+	<ul class="list-group">
+		{% for category, score in grades %}
+			<li class="list-group-item">
+				<div class="row">
+					<div class="col-sm-6 vcenter">
+						<h4>{{category.summary}}</h4>
+						<p>
+							{% if category.description %}
+								{{ category.description }}
+							{% endif %}
+						</p>
+					</div>
+					<div class="col-sm-6 vcenter rating">
+						<div class="inner-rating" >
+							<center>
+								<span class="help-text" style="padding-right: 15px">Bad </span>
+									<input id={{category.summary}} class='slide' data-slider-id='slide' type="text" data-slider-min="1" data-slider-max="5" data-slider-step="1" data-slider-value="{% if not score %}3{% else %}{{ score }}{% endif %}"/>
+								<span class="help-text" style="padding-left: 15px">Good</span>
+							</center>
```

Comment:
Category.summary is a user controlled field, so using input id={{category.summary}} ties the DOM structure to any value they put there, including categories with spaces/random punctuation that browsers might not support.  Something like "category_{{category.id}}" is safer.